static/css/style.css

```diff
@@ -1871,4 +1871,8 @@ section {
 
 h1, h2, h3, h4, h5 {
   color: #37517e;
+}
+
+.footer-contact img {
+  max-width: 20em;
```

Comment:
It may be just how it renders for me, but the 20em leaves little spacing to the 'Useful Links section
![image](https://github.com/user-attachments/assets/250b8452-da97-4c0f-bbd3-1ca12dfdaa8e)
Unless there's a stipulation on the min size, something like 16em leaves more space:
![image](https://github.com/user-attachments/assets/d59f6436-96f2-4f56-94f2-21bb5aef6e86)

Browser width doesn't seem to scale the spacing, so it looks to be safe to change.

This may stem from the areas being 25% width, when maybe they should be 33% as there's only 3 of them?

Reply:
Resolved, updated layout cols for 3 cols (was setup for 4)